### PR TITLE
Instagram and spotify info on profile

### PR DESCRIPTION
**Pull request by rolandbobek**
I filled up the profile page with infos what are in the original tinder app like:
common interests
common friends on facebook
instagram photos
spotify theme track

TODO: set translations in the localization files for new constants
TODO: make a carousel to see instagram photos larger when clicked

### CODE REVIEW

**Comment by mayeaux** [comment]
Awesome, I'll check this out in the morning , thanks a lot!

**Comment by rolandbobek** [comment]
You're welcome!
By the way, by 'carousel' in the TODO section I meant 'lightbox (modal image gallery)'. I'm sorry, it was 3AM.

**Comment by rolandbobek** [comment]
Screenshots of features implemented.

Instagram: the profile picture is clickable and leads to instagram profile. The show more button shows ten more pictures, until the user has more.

Spotify: artists name and song title, album artwork. preview link opens the track in a new window.

Common connections: common facebook friends with name and profile picture.

Common interests: facebook pages you both liked

tinder-feature-1
tinder-feature-2

**Comment by mayeaux** [comment]
Okay I've finally had a chance to check this out, here's my thoughts:

screen shot 2017-01-09 at 2 26 21 pm

I'm a little worried because the instagram photos overflow the visible page, and cause the user to have to scroll. We may want to redesign the frontend a bit so that we use the real estate under the main image, but otherwise the functionality is really great, nicely done.

Common connections was definitely needed, that's perfect.

Why do we only load 30 instagram photos? Would be great if we could do even more.

Also, we can stick Common Interests outside of the main view (stick it in the overflow space) since it's not that big of a deal.

so, tl;dr  : common interests and spotify out of the main view as it's unimportant. maybe we load more instagram photos? also, the clincher for me would be, can we set up the instagram photos such that we can click on one and load it in the main view? that would be a great feature. 

overall top notch work, happy to get this into the codebase!

**Comment by mayeaux** [comment]
Also, let's add this to the profile page so it's not just on the swipe page! 

I'm closing this PR (merged these changes into a branch called 'development', once we get things looking ready for 'prod' there, we can merge it into master)

**Comment by rolandbobek** [comment]
Thanks!
I'm afraid the 30 instagram photo limit comes from tinder itself.
Later I will share my thoughts and ideas to be discussed and then I can implement the changes.
